docs/getting-started.md

```diff
@@ -81,7 +81,10 @@ Flags:
                                   example: 'fec62a97d569662093dbb9739360942f...,31263s5696620s93dbb973d9360942fc2a...'
   -h, --help                      help for scan
   -l, --log-file                  writes log messages to info.log
+  -m, --minimal                   minimal version of results output
+      --no-color                  disable color output
       --no-progress               hides the progress bar
+  -L, --output-lines int          number of lines to be displayed in results output (default 3)
```

Comment:
I would remove the -m and -L short version flags, and just maintain the longer version.

Reply:
resolved